week-2/14-align-self/ignore_this_folder/custom.css

```diff
@@ -0,0 +1,7 @@
+.countries--tall {
```

Comment:
Same thing here with moving the CSS and images into the general directoru, out of the exercise directory.